Ports/edid-decode/package.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=edid-decode
+version=git
+workdir="${port}-master"
+files="https://github.com/a1ive/edid-decode/archive/refs/heads/master.zip edid-decode-master.zip"
```

Comment:
I think one comment got lost - please don't point ports outside of our control to `master`, ever. Just use a commit hash and regularly update that if needed. This then also allows us to use a hash for verification as the download is stable